src/diagnostic-channel-publishers/package.json

```diff
@@ -20,17 +20,20 @@
   },
   "description": "A collection of pre-built module patches that enable existing npm modules to publish diagnostic data",
   "devDependencies": {
+    "@azure/core-tracing": "^1.0.0-preview.5",
+    "@opentelemetry/tracing": "^0.1.1",
     "@types/mocha": "^2.2.40",
     "@types/node": "^10.0.0",
     "@types/pg": "^7.4.11",
-    "diagnostic-channel": "0.2.0",
+    "diagnostic-channel": "file:../diagnostic-channel",
```

Comment:
Is this intentional?

Reply:
Since its a devdep I think this is okay. Ideally I move this repo over to rush/lerna instead of file referencing like this. This PR includes a small change in diagnostic-channel, which requires this to test it quickly